cpu/nrf9160/include/periph_cpu.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2021 Mesotic SAS
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         cpu_nrf9160
+ * @{
+ *
+ * @file
+ * @brief           nRF9160 specific definitions for handling peripherals
+ *
+ * @author          Dylan Laduranty <dylan.laduranty@mesotic.com>
+ */
+
+#ifndef PERIPH_CPU_H
+#define PERIPH_CPU_H
+
+#include "periph_cpu_common.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   System core clock speed, fixed to 64MHz for all NRF9160 CPUs
+ */
+#define CLOCK_CORECLOCK     (64000000U)
```

Comment:
```suggestion
#define CLOCK_CORECLOCK     MHZ(64)
```

You may have to `#include macros/units.h` for this

Reply:
Fixed.